official/resnet/resnet_run_loop.py

```diff
@@ -478,6 +498,7 @@ def resnet_main(
   distribution_strategy = distribution_utils.get_distribution_strategy(
       distribution_strategy=flags_obj.distribution_strategy,
       num_gpus=flags_core.get_num_gpus(flags_obj),
+      num_workers=num_workers,
       all_reduce_alg=flags_obj.all_reduce_alg)
 
   # Creates a `RunConfig` that checkpoints every 24 hours which essentially
```

Comment:
Unrelated to your CL but i just realized we do not distribute evaluation in this case. Should we? Or is it a bad idea because we throw away any partial batch so eval may not be always complete based on the batch size..